### PR TITLE
Removed typo from Instructions

### PR DESCRIPTION
**Pull request by nickBlack4**
Removed the word 'with' which doesn't belong.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

**Review by thecodingaviator — APPROVED** [review]

File: curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/run-functional-tests-on-api-endpoints-using-chai-http.english.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ As a reminder, this project is being built upon the following starter project on
 ## Instructions
 <section id='instructions'>
 Replace <code>assert.fail()</code>. Test the status and the text.response. Make the test pass.
```
Comment (thecodingaviator) [inline curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/run-functional-tests-on-api-endpoints-using-chai-http.english.md]:
```suggestion
Replace <code>assert.fail()</code>. Test the status and the <code>text.response</code>. Make the test pass.
```

**Review by thecodingaviator — CHANGES_REQUESTED** [review]

**Review by nickBlack4 — COMMENTED** [review]
looks good to me!

File: curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/run-functional-tests-on-api-endpoints-using-chai-http.english.md [hunk 2]
```diff
@@ -13,8 +13,8 @@ As a reminder, this project is being built upon the following starter project on
 
 ## Instructions
 <section id='instructions'>
-Replace <code>assert.fail()</code>. Test the status and the text.response. Make the test pass.
-Don't send a name in the query, the endpoint with responds with <code>'hello Guest'</code>.
+Replace <code>assert.fail()</code>. Test the status and the <code>text.response</code>. Make the test pass.
+Don't send a name in the query, the endpoint responds with <code>'hello Guest'</code>.
```
Comment (RandellDawson) [inline curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/run-functional-tests-on-api-endpoints-using-chai-http.english.md]: Do we really need the code tags surrounding the string?
```suggestion
Don't send a name in the query, the endpoint responds with "hello Guest".
```

**Review by RandellDawson — APPROVED** [review]
LGTM  @thecodingaviator What do you think about the recent changes?

**Review by thecodingaviator — APPROVED** [review]
LGTM!

**Comment by thecodingaviator** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**